etc/kayobe/stackhpc.yml

```diff
@@ -166,3 +166,11 @@ stackhpc_docker_registry_password: "{{ pulp_password }}"
 
 # Whether or not to run CIS benchmark hardening playbooks. Default is false.
 #stackhpc_enable_cis_benchmark_hardening_hook:
+
+###############################################################################
+# diskimage-builder elements repos
+stackhpc_dib_image_elements_repos:
+  - repo: "https://github.com/stackhpc/stackhpc-image-elements"
+    local: "{{ source_checkout_path }}/stackhpc-image-elements"
+    version: "rocky-container-pulp-2"
```

Comment:
Use this branch for testing and update when https://github.com/stackhpc/stackhpc-image-elements/pull/67 is merged and a new tag cut.